recipes/xeus/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
We have an example echo kernel that builds and runs. I will add this to the feedstock. I wanted to iterate on this.